algebra/src/mnt4_298/curves/g1.rs

```diff
@@ -0,0 +1,78 @@
+use crate::mnt4_298::{self, Fq, Fr, FR_ONE};
+use algebra_core::{
+    biginteger::BigInteger320,
+    curves::{
+        mnt4,
+        models::{ModelParameters, SWModelParameters},
+    },
+    field_new,
+};
+
+pub type G1Affine = mnt4::G1Affine<mnt4_298::Parameters>;
+pub type G1Projective = mnt4::G1Projective<mnt4_298::Parameters>;
+pub type G1Prepared = mnt4::G1Prepared<mnt4_298::Parameters>;
+
+#[derive(Clone, Default, PartialEq, Eq)]
+pub struct Parameters;
+
+impl ModelParameters for Parameters {
+    type BaseField = Fq;
+    type ScalarField = Fr;
+}
+
+impl SWModelParameters for Parameters {
+    /// COEFF_A = 2
+    #[rustfmt::skip]
+    const COEFF_A: Fq = field_new!(Fq, BigInteger320([
+            3568597988870129848,
+            15257338106490985450,
+            10069779447956199041,
+            5922375556522222383,
+            3858029504390,
+    ]));
+
+    /// COEFF_B = 423894536526684178289416011533888240029318103673896002803341544124054745019340795360841685
```

Comment:
Maybe add a reference to the libff parameters? https://github.com/scipr-lab/libff/blob/c927821ebe02e0a24b5e0f9170cec5e211a35f08/libff/algebra/curves/mnt/mnt4/mnt4_init.cpp#L117